backend/email-service/src/test/java/com/karankumar/bookproject/book/service/EmailServiceTest.java

```diff
@@ -21,9 +21,7 @@
 import com.karankumar.bookproject.configuration.EmailConfiguration;
 import com.karankumar.bookproject.configuration.ThymeleafConfiguration;
 import com.karankumar.bookproject.mock.MockEmailTemplate;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
As per our style guide, we are not using wildcard imports, so could you please undo this?

Reply:
sorry, i'll do it